adapter/kubernetes/kubernetes.go

```diff
@@ -86,9 +86,11 @@ const (
 	serviceVal        = "Service"
 
 	// value extraction
-	clusterDomain        = "svc.cluster.local"
-	podServiceLabel      = "app"
-	istioPodServiceLabel = "istio"
+	clusterDomain                      = "svc.cluster.local"
+	podServiceLabel                    = "app"
+	istioPodServiceLabel               = "istio"
+	lookupIngressSourceAndOriginValues = false
+	istioIngressSvc                    = "ingress.istio-system.svc.cluster.local"
```

Comment:
that doesn't seem to match https://github.com/istio/istio/blob/master/install/kubernetes/istio.yaml#L1108 or is this not the name ? 
what test exercises this ?

Reply:
It does actually match. You just aren't accounting for the mixer matching of service names. It matches through the "istio" label. And, in this case, the "istio" label value is "ingress".

The only tests for this behavior are in this PR itself.

Reply:
istio-ingress.istio-system.svc.cluster.local is the service name
if we have some other lookup it shouldn't look like the dns name svcname.namespace.svc.cluster.local is well defined, where/why are we changing this ?

Reply:
`.svc.cluster.local` is configurable, so it makes sense for the `fully_qualified_istio_ingress_service_name` to be configurable.

Reply:
it is configurable in this PR.  the value here is the default value.